frontend/src/components/user-management/TeacherUserTable.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
The ticket said to create a new TeacherTable but it turned out to be identical to adminTable. Would it be better to reuse AdminTable instead?